docs/developer/release-process.md

```diff
@@ -30,36 +30,27 @@ To create a new release, follow these steps:
+   2. Kick off the [NFR workflow](https://github.com/nginxinc/nginx-gateway-fabric/actions/nfr.yml) in the browser. For `image_tag`, use `edge`, and for `version`, use the upcoming `X.Y.Z` NGF version. This will run all of the NFR tests which are automated and open a PR with the results files when it is complete. Review this PR and make any necessary changes before merging.
 4. If a problem is found, prepare a fix PR, merge it into the main branch and return to the previous step.
 5. If the supported Gateway API minor version has changed since the last release, test NGINX Gateway Fabric with the previous version of the Gateway API CRDs.
 6. If a compatibility issue is found, add a note to the release notes explaining that the previous version is not supported.
 7. Create a release branch following the `release-X.Y` naming convention.
-8. Prepare and merge a PR into the **release branch** to update the repo files for the release:
-    1. Update the Helm [Chart.yaml](/deploy/helm-chart/Chart.yaml): the `appVersion` to `X.Y.Z`, the icon and source
-       URLs to point at `vX.Y.Z`, and bump the `version`.
-    2. Adjust the `VERSION` variable in the [Makefile](/Makefile) and the `TAG` in the
-       [conformance tests Makefile](/conformance/Makefile) to `X.Y.Z`.
-    3. Update the tag of NGF container images used in the Helm [values.yaml](/deploy/helm-chart/values.yaml) file,
-       the [provisioner manifest](/conformance/provisioner/provisioner.yaml), and all docs to `X.Y.Z`.
-    4. Ensure that the `imagePullPolicy` is `IfNotPresent` in the Helm [values.yaml](/deploy/helm-chart/values.yaml)
-       file.
-    5. Generate the installation manifests by running `make generate-manifests`.
-    6. Modify any `git clone` instructions to use `vX.Y.Z` tag.
-    7. Modify any docs links that refer to `main` to instead refer to `vX.Y.Z` or `release-x-y`, whichever is applicable.
-    8. Update any installation instructions to ensure that the supported Gateway API and NGF versions are correct.
-       Specifically, helm README and `site/content/includes/installation/install-gateway-api-resources.md`.
-    9. Update the [README](/README.md) to include information about the release.
-    10. Update the [changelog](/CHANGELOG.md). The changelog includes only important (from the user perspective)
-        changes to NGF. This is in contrast with the autogenerated full changelog, which is created in the next
-        step. As a starting point, copy the important features, bug fixes, and dependencies from the autogenerated
-        draft of the full changelog. This draft can be found under
-        the [GitHub releases](https://github.com/nginxinc/nginx-gateway-fabric/releases) after the release branch is
-        created. Use the previous changelog entries for formatting and content guidance.
+8. Run the [Release PR](./../../.github/workflows/release-pr.yml) workflow to update the repo files for the release. Then there are a few manual steps to complete:
```

Comment:
Is the spacing of this step inconsistent with the rest of the top level steps? It looks a bit off in the diff.

Reply:
I think it had an extra space before?